docs/_includes/common/less.html

```diff
@@ -1,43 +1,169 @@
 <div id="less">
     <h2><span class="secondary-text">@ebay/skin/</span>less</h2>
     <p>The Less module enables developers to access our public Less variables &amp; mixins in their application.</p>
-    <p>Our Less module is <strong>not</strong> intended as a <em>general-purpose</em> library of utility mixins and variables such as <a href="http://lesshat.madebysource.com">LessHat</a>..</p>
+    <p>Our Less module is <strong>not</strong> intended as a <em>general-purpose</em> library of utility mixins and variables such as <a href="http://lesshat.madebysource.com">LessHat</a>.</p>
     {% highlight json %}
 {
     "dependencies": {
       "lasso": "^3",
       "lasso-less": "^3",
-      "@ebay/skin": "^7"
+      "@ebay/skin": "^9"
```

Comment:
Should we think of a way to use a variable to show this so we don't have to always change it. 
I believe there is a `version` variable to use here

Reply:
Well, the version is the full SEMVER version, so I don't know that we want to do that.

Good catch, nonetheless. Any automation of that version number would be good. Maybe we can add this version number to the automation script to update?

Reply:
I'll see if I can use the version variable. Cheers.

Reply:
Jekyll/Liquid has a function to do what we need:

https://shopify.github.io/liquid/filters/slice/

Reply:
Oh, nice. I didn't know about the base for Jekyll being Liquid. Good to know!

Reply:
Future fix?

Reply:
Fixed. Should be at least.

Reply:
Ohhh, I changed it in install.html. Let me do less.html too.

Reply:
Oh, I see the slice above, in the install.html file.